inspirehep/modules/workflows/workflows/edit_article.py

```diff
@@ -41,6 +47,19 @@ def update_record(obj, eng):
     record.update(obj.data)
     record.commit()
 
+    transaction_id = None
+    record = get_db_record('lit', control_number)
+    for revision in record.revisions:
+        if revision.revision_id == record.revision_id:
+            transaction_id = revision.model.transaction_id
+            break
+
+    Transaction = transaction_class(RecordMetadata)
+    transaction = Transaction.query.filter(Transaction.id == transaction_id).one()
+    transaction.user_id = obj.id_user
+    db.session.add(transaction)
+    db.session.commit()
```

Comment:
I'd move the transaction logic to separate function so you can easily test it, now you have to write the code https://github.com/inspirehep/inspire-next/pull/3802/files#diff-e05ba1f919990d666d23d8a40efd95e2R196

Reply:
well this is just an early thing, I'll separate things later 👍